src/components/SearchCard.tsx

```diff
@@ -125,15 +130,15 @@ export const SearchCard = forwardRef<HTMLDivElement, SearchCardProps>(
             ) : null}
 
             <Stack
-              direction={winWidth < 900 || sm ? 'row-reverse' : 'row'}
+              direction={medium || mapCard ? 'row-reverse' : 'row'}
               alignItems="end"
-              justifyContent={winWidth < 900 || sm ? 'start' : 'space-between'}
+              justifyContent={'space-between'}
             >
               <Stack
-                direction={winWidth < 900 || sm ? 'column' : 'row'}
-                alignItems={winWidth < 900 || sm ? 'end' : 'center'}
+                direction={medium || mapCard ? 'column' : 'row'}
```

Comment:
same as the other comment about responsiveness

Reply:
I would personallly stick as much as possible to the way MUI recommends us to use responsiveness in styling properties. This  prevents naming inconsistencies (e.g. you call it `medium` here but `isMobile` on a different page.

``` direction={{ xs: 'column', md: mapCard ? 'column' : 'row}}```

ofc you sometime have to use the `useMediaQuery` hook like you are doing here, but I would not do that for styling . Only to include/exclude a component on certain screen sizes.